groovy/inductor_dashboard.groovy

```diff
@@ -0,0 +1,361 @@
+    if (params.THREAD != '') {
+        THREAD = params.THREAD
+    }
+}
+echo "THREAD: $THREAD"
+
+CHANNELS = ''
+if ('CHANNELS' in params) {
+    echo "CHANNELS in params"
+    if (params.CHANNELS != '') {
+        CHANNELS = params.CHANNELS
+    }
+}
+echo "CHANNELS: $CHANNELS"
+
+MODEL_SUITE = ''
+if ('MODEL_SUITE' in params) {
+    echo "MODEL_SUITE in params"
+    if (params.MODEL_SUITE != '') {
+        MODEL_SUITE = params.MODEL_SUITE
+    }
+}
+echo "MODEL_SUITE: $MODEL_SUITE"
+
+inductor_tools_branch = ''
+if ('inductor_tools_branch' in params) {
+    echo "inductor_tools_branch in params"
+    if (params.inductor_tools_branch != '') {
+        inductor_tools_branch = params.inductor_tools_branch
+    }
+}
+echo "inductor_tools_branch: $inductor_tools_branch"
+
+def get_time(){
+    return new Date().format('yyyy-MM-dd')
+}
+env._VERSION = get_time()
+println(env._VERSION)
+
+env._NODE = "$NODE_LABEL"
+
+node(NODE_LABEL){
+    checkout scm
+    stage("get image and inductor-tools repo"){
+        echo 'get image and inductor-tools repo......'
+        if ("${Build_Image}" == "true") {
+            def image_build_job = build job: 'inductor_images', propagate: false, parameters: [
+                [$class: 'StringParameterValue', name: 'NODE_LABEL', value: 'Docker'],
+                [$class: 'StringParameterValue', name: 'PT_REPO', value: "${PT_REPO}"],
+                [$class: 'StringParameterValue', name: 'PT_BRANCH', value: "${PT_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'PT_COMMIT', value: "${PT_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_VISION_BRANCH', value: "${TORCH_VISION_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_VISION_COMMIT', value: "${TORCH_VISION_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_TEXT_BRANCH', value: "${TORCH_TEXT_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_TEXT_COMMIT', value: "${TORCH_TEXT_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_DATA_BRANCH', value: "${TORCH_DATA_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_DATA_COMMIT', value: "${TORCH_DATA_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_AUDIO_BRANCH', value: "${TORCH_AUDIO_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_AUDIO_COMMIT', value: "${TORCH_AUDIO_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_BENCH_BRANCH', value: "${TORCH_BENCH_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_BENCH_COMMIT', value: "${TORCH_BENCH_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'BENCH_COMMIT', value: "${BENCH_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'inductor_tools_branch', value: "${inductor_tools_branch}"],
+                [$class: 'StringParameterValue', name: 'tag', value: 'nightly'],
```

Comment:
We don't need hard code for the image `tag`. Also use a parameter for it with "nightly" default value